src/md/winmd/winmdimport.cpp

```diff
@@ -1654,15 +1656,17 @@ class WinMDImport : public IMetaDataImport2
             // Step 1: Call EnumAssemblyRefs with an empty buffer to create the HENUMInternal
             IfFailGo(m_pRawAssemblyImport->EnumAssemblyRefs(phEnum, NULL, 0, NULL));
 
-            // Step 2: Increment the cound to include the extra assembly refs
-            HENUMInternal *phInternalEnum = static_cast<HENUMInternal*>(*phEnum);
+            {
+                // Step 2: Increment the cound to include the extra assembly refs
```

Comment:
Since you are touching this file, could you please fix the typo here? cound -> count